src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,12 +26,39 @@ public RobotContainer() {
     configureBindings();
   }
 
-  private void configureBindings() {
-    drivetrainBase.setDefaultCommand(
-        new AttackJoystickDriveControl(drivetrainBase, leftJoystick, rightJoystick));
-    shooterBase.setDefaultCommand(
-        new AttackJoystickShooterControl(shooterBase, leftJoystick, rightJoystick));
-    intakeBase.setDefaultCommand(
-        new AttackJoystickIntakeControl(intakeBase, leftJoystick, rightJoystick));
+  private void configureBindings(OperatorModes operatorMode) {
+    switch(operatorMode) {
+      case ATTACK_ONLY:
+        drivetrainBase.setDefaultCommand(
+            new AttackJoystickDriveControl(drivetrainBase, leftJoystick, rightJoystick));
+        shooterBase.setDefaultCommand(
+            new AttackJoystickShooterControl(shooterBase, leftJoystick, rightJoystick));
+        intakeBase.setDefaultCommand(
+            new AttackJoystickIntakeControl(intakeBase, leftJoystick, rightJoystick));
+        break;
+      case XBOX_ONLY: 
+        drivetrainBase.setDefaultCommand(new XboxJoystickDriveControl(drivetrainbase, xboxController))
+        break;
+      case XBOX_AND_ATTACK:
+        drivetrainBase.setDefaultCommand(new XboxJoystickDriveControl(drivetrainbase, xboxController))
+        break;
+    }
   }
+  
+  private void configureButtonBindings() {
```

Comment:
this is devil code, I removed it once I saw my buffoonery. Manually setting controls on boot is better as 90% of the time, this will not work